src/software/simulation/er_force_simulator.cpp

```diff
@@ -226,7 +238,7 @@ std::vector<SSLProto::SSL_WrapperPacket> ErForceSimulator::getSSLWrapperPackets(
 
 Field ErForceSimulator::getField() const
 {
-    return Field::createSSLDivisionAField();
+    return field;
```

Comment:
please add a test for this function, i.e. create an ErForceSimulator with DIV_A/DIV_B and check the Field returned by getField.